2021/25xxx/CVE-2021-25657.json

```diff
@@ -1,18 +1,110 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2021-25657",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.9"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2021-25657",
+    "ASSIGNER": "securityalerts@avaya.com",
+    "DATE_PUBLIC": "",
+    "TITLE": "Avaya IP Office Privilege Escalation Vulnerability",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [],
+    "advisory": "ASA-2022-114",
+    "discovery": "EXTERNAL"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Avaya",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "IP Office",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "0",
+                      "version_affected": "<=",
+                      "version_value": "11.1 Feature Pack 2 Service Pack 1",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-269"
+          }
         ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "A privilege escalation vulnerability was discovered in Avaya IP Office Admin Lite and USB Creator that may potentially allow a local user to escalate privileges. This issue affects Admin Lite and USB Creator 11.1 Feature Pack 2 Service Pack 1 and earlier versions."
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://support.avaya.com/css/P8/documents/101083319",
+        "name": ""
```

Comment:
Please remove the Name attribute here if you are leaving it blank. Once that change happens we should be able to process that request.

Reply:
Removed. Thanks.